documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -143,7 +150,8 @@ The date MAY optionally be followed by one of:
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="M3016" />M3016 PropertiesNamesCamelCase
+### <a name="R3014" />R3014 BodyPropertiesNamesCamelCase
+### <a name="R3016" />R3016 DefinitionsPropertiesNamesCamelCase
```

Comment:
I can do that if it'd be better, do you think it'll be better to repeat the content though? I'm wondering, since updating one, means we need to repeat changes in the other one?